ntbs-service/DataAccess/AlertRepository.cs

```diff
@@ -49,6 +49,7 @@ public async Task<IList<Alert>> GetAlertsByTbServiceCodesAsync(IEnumerable<strin
         {
             return await GetBaseAlertIQueryable()
                 .Where(a => tbServices.Contains(a.TbServiceCode))
+                .Where(n => n.Notification.NotificationStatus != NotificationStatus.Draft)

```

Comment:
The `n` variable probably ought to be a `a`?
Also, not all alerts have a notification, will this throw a NPE in those cases?

Reply:
Good point, not a NPE but just doesn't get fetched! Will fix